android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -507,6 +517,18 @@ protected void onDestroyInternal() {
         cleanUpMiscAndroidMetrics();
     }
 
+    @Override
+    public void performOnConfigurationChanged(Configuration newConfig) {
+        super.performOnConfigurationChanged(newConfig);
+
+        Tab currentTab = getActivityTab();
```

Comment:
this code also doesn't seem like it makes sense here, seems like there should be a separate activity for youtube?

Reply:
or should this be an activity at all or a Coordinator that implements ConfigurationChangedObserver?